### ISSUE TITLE
[Docs] - Clarify multi collector

### ISSUE THREAD

**Issue by GJBoth** [pytorch/rl#2123]
None of the docs seem to mention (at least, where I could find it), that for the `MultiSyncCollector` (and async) you need to run your code using:

```
if __name__ == "__main__":
    training_loop()
```

otherwise, you'll get this error:
```
RuntimeError: 
        An attempt has been made to start a new process before the
        current process has finished its bootstrapping phase.

        This probably means that you are not using fork to start your
        child processes and you have forgotten to use the proper idiom
        in the main module:

            if __name__ == '__main__':
                freeze_support()
                ...

        The "freeze_support()" line can be omitted if the program
        is not going to be frozen to produce an executable.

        To fix this issue, refer to the "Safe importing of main module"
        section in https://docs.python.org/3/library/multiprocessing.html
```

**Comment by vmoens** [comment]
Fair point! We should mention that

**Comment by GJBoth** [comment]
I'll prep a PR - hopefully today, if not early next week :-).

**Comment by GJBoth** [comment]
Closed by #2126
